docs/install.md

```diff
@@ -0,0 +1,72 @@
+# Installing BKPR
+
+A Bitnami Kubernetes Production Runtime (BKPR) release consists of a collection of Kubernetes manifests written in Jsonnet plus the accompanying `kubeprod` installer binary. The kubeprod binary deals with all the platform-specific details, evaluates the Jsonnet manifests, and applies them to the existing Kubernetes cluster.
+
+This document walks you through installing the `kubeprod` binary.
+
+## Installing the `kubeprod` installer
+
+The `kubeprod` binary can be installed from pre-built binary releases or be built directly from source.
+
+### Installing from binary releases
+
+BKPR releases are available for 64-bit versions of Linux, macOS and Windows platforms. Download the latest stable version for your platform of choice from the [releases](https://github.com/bitnami/kube-prod-runtime/releases) page.
+
+For convenience lets define the `BKPR_VERSION` environment variable:
+
+```bash
+BKPR_VERSION=$(curl --silent "https://api.github.com/repos/bitnami/kube-prod-runtime/releases/latest" | jq -r '.tag_name')
+```
+
+_The command configures the `BKPR_VERSION` variable with the latest stable version of BKPR. If you wish to use a pre-release or a specific version, please set it up accordingly._
+
+* Step 1: Use the following commands to download the desired release:
+
+On Linux:
+
+  ```bash
+  curl -LO https://github.com/bitnami/kube-prod-runtime/releases/download/${BKPR_VERSION}/bkpr-${BKPR_VERSION}-linux-amd64.tar.gz
+  tar xf bkpr-${BKPR_VERSION}-linux-amd64.tar.gz
+  ```
+
+On macOS:
+
+  ```bash
+  curl -LO https://github.com/bitnami/kube-prod-runtime/releases/download/${BKPR_VERSION}/bkpr-${BKPR_VERSION}-darwin-amd64.tar.gz
+  tar xf bkpr-${BKPR_VERSION}-darwin-amd64.tar.gz
+  ```
+
+* Step 2: Install the `kubeprod` with:
+
+  ```bash
+  chmod +x bkpr-${BKPR_VERSION}/kubeprod
+  sudo mv bkpr-${BKPR_VERSION}/kubeprod /usr/local/bin/
+  ```
+
+Note: The Jsonnet manifests from the release are used while installing BKPR to the cluster.
+
+### Building from source (Linux / macOS)
+
+To build the `kubeprod` binary from source you need the [Go](https://golang.org/) compiler. Please follow the [Go install guide](https://golang.org/doc/install) to install the compiler on your machine before proceeding with this section.
+
+* Step 1: Set up the Go environment variables:
+
+  ```bash
+  export GOPATH=$HOME/go
+  export PATH=$GOPATH/bin:$PATH
+  ```
+
+* Step 2: Build `kubeprod`:
+
+  ```bash
+  go get github.com/bitnami/kube-prod-runtime/kubeprod
+  ```
+  
+  The `kubeprod` binary will be installed at `$GOPATH/bin/kubeprod`. 
```

Comment:
In the previous "install from binary" section, you moved the file to /usr/local/bin so it would be in the path. Is it not necessary to do that here?

Reply:
Since we're installing from source here the user will end up installing a development version on their machine. For this reason I refrained from advising the user to move the compiled binary to the system path.